cli/internal/cache/cache_http_test.go

```diff
@@ -182,7 +182,7 @@ func TestRestoreTar(t *testing.T) {
 
 	expectedFiles := []turbopath.AnchoredSystemPath{
 		turbopath.AnchoredUnixPath("extra-file").ToSystemPath(),
-		turbopath.AnchoredUnixPath("my-pkg/").ToSystemPath(),
+		turbopath.AnchoredUnixPath("my-pkg").ToSystemPath(),
```

Comment:
This output is only consumed inside of the `taskSummary`. This change effectively aligns FS and HTTP cache outputs.